packages/intl-locale/index.ts

```diff
@@ -639,41 +639,103 @@ export class Locale {
     return parseUnicodeLanguageId(locale).region
   }
 
+  /**
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getCalendars
+   * https://tc39.es/proposal-intl-locale-info/#sec-Intl.Locale.prototype.getCalendars
+   */
+  public getCalendars() {
+    return calendarsOfLocale(this)
+  }
+
   /**
    * https://tc39.es/proposal-intl-locale/#sec-Intl.Locale.prototype.calendars
    */
   public get calendars() {
     return calendarsOfLocale(this)
   }
 
+  /**
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getCollations
+   * https://tc39.es/proposal-intl-locale-info/#sec-Intl.Locale.prototype.getCollations
+   */
+  public getCollations() {
+    return collationsOfLocale(this)
+  }
+
   /**
    * https://tc39.es/proposal-intl-locale/#sec-Intl.Locale.prototype.collations
    */
   public get collations() {
     return collationsOfLocale(this)
   }
 
+  /**
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getHourCycles
+   * https://tc39.es/proposal-intl-locale-info/#sec-Intl.Locale.prototype.getHourCycles
+   */
+  public getHourCycles() {
+    return hourCyclesOfLocale(this)
```

Comment:
missing `initializedLocale` check per spec

Reply:
Same as the comment above: `hourCyclesOfLocale` is using internally `getInternalSlots` so I'm not sure where I should add this `initializedLocale` check

<img width="450" alt="image" src="https://github.com/formatjs/formatjs/assets/22725671/0ca83e23-6933-4c8a-9bfc-0d24cc0e20bb">

Reply:
Added 👍